modules/op-release-notes-1-10.adoc

```diff
@@ -228,4 +228,32 @@ With this update, {pipelines-title} General Availability (GA) 1.10.5 is availabl
 
 * Before this update, huge pipeline runs were not getting listed or deleted using the `oc` and `tkn` commands. This update mitigates this issue by compressing the huge annotations that were causing this problem. Remember that if the pipeline runs are still too huge after compression, then the same error still recurs.
 
-* Before this update, only the pod template specified in the `pipelineRun.spec.taskRunSpecs[].podTemplate` object would be considered for a pipeline run. With this update, the pod template specified in the `pipelineRun.spec.podTemplate` object is also considered and merged with the template specified in the `pipelineRun.spec.taskRunSpecs[].podTemplate` object.
\ No newline at end of file
+* Before this update, only the pod template specified in the `pipelineRun.spec.taskRunSpecs[].podTemplate` object would be considered for a pipeline run. With this update, the pod template specified in the `pipelineRun.spec.podTemplate` object is also considered and merged with the template specified in the `pipelineRun.spec.taskRunSpecs[].podTemplate` object.
+
+[id="release-notes-1-10-6_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.6
+
+With this update, {pipelines-title} General Availability (GA) 1.10.6 is available on {OCP} 4.10 in addition to 4.11, 4.12, and 4.13.
+
+This update includes an updated version of the `tkn` command line tool. You can download the updated version of this tool at the following locations:
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.10.6/tkn-linux-amd64.tar.gz[Linux (x86_64, amd64)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.10.6/tkn-linux-s390x.tar.gz[Linux on {ibmzProductName} and {linuxoneProductName} (s390x)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.10.6/tkn-linux-ppc64le.tar.gz[Linux on {ibmpowerProductName} (ppc64le)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.10.6/tkn-linux-arm64.tar.gz[Linux on ARM (aarch64, arm64)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.10.6/tkn-windows-amd64.zip[Windows]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.10.6/tkn-macos-amd64.tar.gz[macOS]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.10.6/tkn-macos-arm64.tar.gz[macOS on ARM]
+
+If you installed the `tkn` command line tool using RPM on {op-system-base-full}, use the `yum update` command to install the updated version.
+
+[id="fixed-issues-1-10-6_{context}"]
+=== Fixed issues
+
+* With this update, the S2I cluster task uses a General Availability container image.
```

Comment:
It doesn't look like you link to fixed issues, but it might be something to consider so the customer can go and look at the Jira if they wish.